src/profile/profile.service.ts

```diff
@@ -97,6 +100,14 @@ export class ProfileService {
    */
   async deleteProfile(profileId: string): Promise<void> {
     try {
+      const profileAnalytics = await this.analyticsLogRepo.findOne({
```

Comment:
why is this findOne?? @johnayinde

Reply:
It finds by id analytic log belonging to the profile